### DISCUSSION TITLE
QuML questions publish workflow in workspace

### DISCUSSION THREAD

**Discussion by Krishnaj20** [Sunbird-coKreat/Community#51] [Feature Requests]
Please go thru this thread: https://github.com/orgs/Sunbird-Ed/discussions/382

In the current implementation of workspace, the QuML questions do not appear to the reviewer for review and publish unlike other assets. This capability needs to be addressed to expedite the adoption of inQuiry building block.

@surendrasinghs @@mmanimegalai @vinukumar-vs

**Comment by Krishnaj20** [comment]
@mmanimegalai - let us know if this item is part of coKreat roadmap?

**Reply by surendrasinghs** [reply]
+1 to the suggestion - Workspace should support Question Set objects to enable complete lifecycle (draft, review, publish, etc). Thanks, Krishna.

@poojajmanjooran @pallakartheekreddy Please track this closely.
